lib/Dashboard/Model/Jobs.pm

```diff
@@ -31,6 +31,26 @@ sub add ($self, $job) {
   );
 }
 
+sub delete_job ($self, $id) {
+  $self->pg->db('DELETE from openqa_jobs where job_id=?', $id);
+}
+
+# called from startup in AMQP watcher to catch up with jobs started/ended while the watcher wasn't alive
+sub fetch_old ($self) {
```

Comment:
This is duplicate code from `lib/Dashboard/Model/AMQP.pm`, right?

Reply:
I moved the code forth and back as I couldn't find a sweet spot between the models dependencies. The code in AMQP is dead code, I'll remove that